boards/frdm-k64f/include/board.h

```diff
@@ -53,6 +53,20 @@ extern "C"
 #define LED2_TOGGLE        (GPIOB->PTOR = LED2_MASK)
 /** @} */
 
+/**
+ * @name    Button pin definitions
+ * @{
+ */
+/* SW2, SW3 will short these pins to ground when pushed. PTA4 has an external
+ * pull-up resistor to VDD, but there is no external pull resistor on PTC6 */
```

Comment:
From the schematic, there's also a pull resistor on PCT6. Are you sure of you are writing here ? (I don't have this hardware for testing unfortunately).
I would have put GPIO_IN_PU to both button modes.

Reply:
One of the pull resistors are DNP (do not populate) in the schematic, probably to avoid affecting the INT1 pin on the accelerometer.
I could change both to GPIO_IN_PU if you prefer.

Reply:
@aabadie the link you posted is for the frdm-kw41z, this PR is for the frdm-k64f

Reply:
I will change the mode to IN_PU on both

Reply:
ooops, not fully awake..

Reply:
Ok, in k64f schematics, both buttons pull resistors are DNP. So ok for IN_PU.

Reply:
I noticed that the R75 resistor is only populated in revision E3 boards or later. (Change log in E4 schematic states: "E3:  Depopulating C55, populating R75"). Using IN_PU should probably be the most robust solution here.